modules/tooltip/react/stories/stories.tsx

```diff
@@ -1,76 +1,130 @@
+};
 
-class TooltipWrapper extends React.Component<{}, TooltipWrapperState> {
-  private tooltipRef: React.RefObject<HTMLDivElement>;
-  public constructor(props: {}) {
-    super(props);
-    this.tooltipRef = React.createRef();
-    this.state = {
-      open: false,
-      anchorEl: null,
-    };
-  }
+export const Default = () => {
+  return (
+    <Tooltip title="Close">
+      <IconButton variant={IconButton.Variant.Circle} icon={xIcon} aria-label="Close" />
+    </Tooltip>
+  );
+};
 
-  public render() {
-    const {open} = this.state;
-    return (
-      <div>
-        <h3>Hover over the icon.</h3>
-        <div
-          style={{display: 'inline-flex'}}
-          ref={this.tooltipRef}
-          onMouseEnter={this.open}
-          onMouseLeave={this.close}
-          onFocus={this.open}
-          onBlur={this.close}
-        >
-          <IconButton
-            variant={IconButton.Variant.Circle}
-            icon={xIcon}
-            aria-labelledby={'123'}
-            aria-label="Close"
-          />
+export const DescribeType = () => {
+  return (
+    <Tooltip type="describe" title="The service will restart after this action">
+      <DeleteButton>Delete</DeleteButton>
+    </Tooltip>
+  );
+};
+
+export const CustomContent = () => {
+  return (
+    <Tooltip
+      type="describe"
+      title={
+        <div>
+          This is a <em>custom</em> tooltip with <strong>custom HTML</strong>.
+          <br />
+          Any HTML is valid here.
+          <br />
+          Keep it short and note that assistive technology will read this content as text-only and
+          maybe have verbosity turned off to the point where this content is completely ignored.
+          Consider a different way to describe important content.
         </div>
-        <Popper open={open} anchorElement={this.state.anchorEl} placement={'top'}>
-          <Tooltip id={'123'}>Close</Tooltip>
-        </Popper>
-      </div>
-    );
-  }
+      }
+    >
+      <Button>Hover Me</Button>
+    </Tooltip>
+  );
+};
+
+export const UseTooltip = () => {
+  const {targetProps, popperProps, tooltipProps} = useTooltip();
+
+  return (
+    <React.Fragment>
+      <h3>Hover over the icon</h3>
+      <IconButton
+        variant={IconButton.Variant.Circle}
+        icon={xIcon}
+        {...targetProps}
+        aria-label="Close"
+      />
+      <Popper placement="top" {...popperProps}>
+        <TooltipContainer {...tooltipProps}>Close</TooltipContainer>
+      </Popper>
+    </React.Fragment>
+  );
+};
 
-  private close = () => {
-    this.setState({
-      open: false,
-    });
+export const Placements = () => {
+  const placementStyles: CSSObject = {
+    display: 'flex',
+    justifyContent: 'space-around',
   };
 
-  private open = () => {
-    this.setState({
-      open: true,
-      anchorEl: this.tooltipRef.current,
-    });
+  const createPlacement = (placement: Placement) => {
+    return (
+      <Tooltip title="Add" placement={placement}>
```

Comment:
Consider using HTML here to show that HTML works inside tooltips

Reply:
I reviewed this, but the visual story uses `TooltipContainer` instead of `Tooltip`. I don't think adding HTML to that story will give us much confidence unless we add configuration to `Tooltip` to show it open all the time.